packages/react-charts/src/components/ChartTheme/themes/dark/multi-color-ordered-theme.ts

```diff
@@ -29,31 +29,31 @@ import { ColorTheme } from '../color-theme';
 // The color order below (minus the purple color family) improves the color contrast in ordered charts; donut, pie, bar, & stack
 // See https://docs.google.com/document/d/1cw10pJFXWruB1SA8TQwituxn5Ss6KpxYPCOYGrH8qAY/edit
 const COLOR_SCALE = [
-  chart_color_blue_300.value,
-  chart_color_green_300.value,
-  chart_color_cyan_300.value,
-  chart_color_gold_300.value,
-  chart_color_orange_300.value,
-  chart_color_blue_100.value,
-  chart_color_green_500.value,
-  chart_color_cyan_100.value,
-  chart_color_gold_100.value,
-  chart_color_orange_500.value,
-  chart_color_blue_500.value,
-  chart_color_green_100.value,
-  chart_color_cyan_500.value,
-  chart_color_gold_500.value,
-  chart_color_orange_100.value,
-  chart_color_blue_200.value,
-  chart_color_green_400.value,
-  chart_color_cyan_200.value,
-  chart_color_gold_200.value,
-  chart_color_orange_400.value,
-  chart_color_blue_400.value,
-  chart_color_green_200.value,
-  chart_color_cyan_400.value,
-  chart_color_gold_400.value,
-  chart_color_orange_200.value
+  chart_color_blue_300.var,
+  chart_color_green_300.var,
+  chart_color_cyan_300.var,
+  chart_color_gold_300.var,
+  chart_color_orange_300.var,
+  chart_color_blue_100.var,
+  chart_color_green_500.var,
+  chart_color_cyan_100.var,
+  chart_color_gold_100.var,
+  chart_color_orange_500.var,
+  chart_color_blue_500.var,
+  chart_color_green_100.var,
+  chart_color_cyan_500.var,
+  chart_color_gold_500.var,
+  chart_color_orange_100.var,
+  chart_color_blue_200.var,
+  chart_color_green_400.var,
+  chart_color_cyan_200.var,
+  chart_color_gold_200.var,
+  chart_color_orange_400.var,
+  chart_color_blue_400.var,
+  chart_color_green_200.var,
+  chart_color_cyan_400.var,
+  chart_color_gold_400.var,
+  chart_color_orange_200.var
```

Comment:
Michael Celedonia suggested he was having trouble with color pallets and the dark them. I understood he wanted to shift the multi-color pallet by one color.

Are we not going to create vars for all color pallets? 

I see that `chart_bullet_negative_measure_ColorScale_100.var` was used for the bullet chart